content/guides/booting.md

```diff
@@ -99,3 +99,31 @@ Adding Haiku to your [GRUB Legacy](http://www.gnu.org/software/grub/grub-legacy.
     title Haiku
     root (hd0,2)
     chainloader +1
+
+### Adding Haiku to the Windows bootloader using EasyBCD
+
+* First get EasyBCD from [this site](https://neosmart.net/EasyBCD/)
+* Install it as you would install any windows software
+* When you launch it you'll see 2 panels
+* On the left panel click on "Add new entry"
+
+[](/files/guides/easybcd_0.png)
```

Comment:
Shouldn't this (and following) be: `![](/files/guides/easybcd_0.png)`
You could also add an actual title in the `[]` and the images have an all-capital .PNG suffix.